### ISSUE TITLE
Android Basics: Fragments Navigation component

### ISSUE THREAD

**Issue by borisdoescode** [google-developer-training/android-basics-kotlin-words-app#83]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-training-fragments-navigation-component

**In which task and step of the codelab can this issue be found?**
Task 4 and after step 3.

**Describe the problem**
The `tools:context` attribute for both files is being set to .WordListFragment

**Comment by danielslabo** [comment]
Duplicate of #48

**Comment by jtavva** [comment]
@borisdoescode 
Thank you so much for reporting this issue. This is fixed and will be published soon.
